eks_infrastructure/env_setup.sh

```diff
@@ -3,11 +3,12 @@
 
 set -ex
 
-# The below url/version is based on EKS v1.20. The same needs to be updated for EKS version upgrade.
-KUBECTL_CLIENT="https://amazon-eks.s3-us-west-2.amazonaws.com/1.20.4/2021-04-12/bin/linux/amd64/kubectl"
+# The below url/version is based on EKS v1.25.6. The same needs to be updated for EKS version upgrade.
+KUBECTL_CLIENT="https://s3.us-west-2.amazonaws.com/amazon-eks/1.25.6/2023-01-30/bin/linux/amd64/kubectl"
 EKSCTL_CLIENT="https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz"
-AWS_IAM_AUTHENTICATOR="https://amazon-eks.s3-us-west-2.amazonaws.com/1.20.4/2021-04-12/bin/linux/amd64/aws-iam-authenticator"
-LATEST_KUBECTL_CLIENT_VERSION=1.20
+AWS_IAM_AUTHENTICATOR="https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v0.6.2/aws-iam-authenticator_0.6.2_linux_amd64"
+
+LATEST_KUBECTL_CLIENT_VERSION=1.25.6
```

Comment:
nit: expected to be just major.minor

Reply:
I don't think it is needed. See open PR that I created to remove it https://github.com/aws/deep-learning-containers/pull/827

Let's remove it and see if the test are green . This also applies to other components added in the file that are not directly linked with training operators.